### DISCUSSION TITLE
Release v3.1.2

### DISCUSSION THREAD

**Discussion by miurahr** [miurahr/aqtinstall#653] [Announcements]
Support WASM for Qt 6.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/miurahr/aqtinstall/releases/tag/v3.1.2'>Release v3.1.2</a>.</em>

**Comment by rabits** [comment]
Hello @miurahr , when we can expect the new release? For now I can't properly build the macos image because the fixes are only in the master branch: https://github.com/state-of-the-art/qt6-docker/issues/3#issuecomment-1498528643

**Comment by miurahr** [comment]
you can use v3.1.5 that is published in pypi in end of march.

**Reply by rabits** [reply]
Oh you right, sorry missed it under github releases) Yeah, v3.1.5 works well)
